Assets/HoloToolkit/Utilities/Scripts/StartAwareSingleton.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
StartAwareSingleton isn't necessary for input to work, but it is analogous to StartAwareBehaviour which is already in HoloToolkit, so I feel we should take this addition.  It was very useful when making our app.  Anyone feel strongly that we shouldn't?